test/wasmfs/wasmfs_fetch.js

```diff
@@ -1,5 +1,7 @@
 addToLibrary({
   getUrlOrigin: function (ptr, len) {
+    ptr = Number(ptr);
+    ptr = Number(len);
```

Comment:
Should this not be closer to where the pointer arrives from wasm, and use the usual non-manual macros to convert as needed?

Reply:
You are right.. I'm not sure why it didn't just use `__sig` here.